modules/standard/Barrier.chpl

```diff
@@ -19,7 +19,7 @@
 
 /* Support for task barriers.
 
-   The Barrier records provided in this module can be used to prevent tasks
+   The Barrier record provided in this module can be used to prevent tasks
```

Comment:
You could optionally use 'type' here rather than 'record' to make it agnostic to whether we use a record or a class in the longer-term.  I think this should be 'class' otherwise?